initfiles/bash/etc/systemd/system/CMakeLists.txt

```diff
@@ -32,7 +32,7 @@ set(componentList
     backupnode
     toposerver)
 
-set(dafilesrv "dafilesrv" "-L ${LOG_PATH} -I %i -D" "")
+set(dafilesrv "dafilesrv" "--logDir=${LOG_PATH} --name=%i --daemon" "")
```

Comment:
This is correct. This should target 8.4.x I believe. The change to the Usage for dafilesrv happened in https://github.com/hpcc-systems/HPCC-Platform/commit/1bc5da6d3e1e55aa37819b0f904d702c9b501fde#.  

Also note; the --daemon call here does not require an instanceName after it because dafilesrv is using its own code to fork as a daemon, not the code in jutil.